### PR TITLE
Chore/fix builds osrm

### PR DESCRIPTION
**Pull request by irony**

### CODE REVIEW

File: k8s/dependencies/pelias/api.yaml [hunk 1]
```diff
@@ -40,6 +40,12 @@ spec:
             - name: PELIAS_CONFIG
               value: "/etc/config/pelias.json"
           resources:
+<<<<<<< HEAD
```
Comment (StefanWallin) [inline k8s/dependencies/pelias/api.yaml:43]: This shouldn't be here!

File: k8s/dependencies/pelias/api.yaml [hunk 2]
```diff
@@ -70,7 +76,10 @@ apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   name: pelias-api-ingress
+<<<<<<< HEAD
```
Comment (StefanWallin) [inline k8s/dependencies/pelias/api.yaml:79]: Or this!

File: k8s/dependencies/pelias/config.yaml [hunk 3]
```diff
@@ -29,7 +29,11 @@ data:
         "autocomplete": {
           "exclude_address_length": 0
         },
+<<<<<<< HEAD
```
Comment (StefanWallin) [inline k8s/dependencies/pelias/config.yaml:32]: Or this!.. and more below!

File: k8s/dependencies/pelias/api.yaml (hunk 1 above)
Comment (irony) [reply k8s/dependencies/pelias/api.yaml:43]: no! I know ;)
I'm trying to remove this pull request and start over.

**Pull request closed without merging**